### DISCUSSION TITLE
can't isolate gpus on windows and ton mining doesn't work

### DISCUSSION THREAD

**Discussion by eldritchdeli** [Lolliedieb/lolMiner-releases#1046] [Q&A]
On windows I tired isolating my 3060 using the --devices arg0 code. It doesn't work on dual mining bat files but on everything else. What makes this worse is that I'm trying to test the dual mining on my 3060 and 1660s together but I think the code can only work with the same gpu as it currently just mines eth not ton or just fails.

**Comment by mohsenk94** [comment]
well, Since I am using 10 out of my 11GPU rig on LOL 1.43 now I can tell you it's working fine on ETH+TON
what you need to do is you need to list your devices and say if your 3060 is GPU0 , you need to add this command to your Miner:
--devices 0
or
-d 0
for short

**Reply by donPuerto** [reply]
do we have bat file example. ie how to overclock?

**Reply by mohsenk94** [reply]
> 

I don't totally understand what you mean exactly, but however this is my bat file:


```
set "POOL=eu1.ethermine.org:4444"
set "WALLET=MyETHWallet"
set "TONPOOL=https://next.ton-pool.com"
set "TONWALLET=MyTonWallet"									

set "DEVICES=0,1,2,3,4,5,6,7,8,9,10"
set "CCLOCK=1300,1300,1500,1400,1300,1600,1300,1400,1300,1600,1400"
set "EXTRAPARAMETERS=--dns-over-https 0"
```

Cards are in this order:
3070TI,3070TI,2060Super,3060TI,3070TI,3080,3060TI,3080,3070TI,3080,3080TI


I didn't have success with --mclk switch to force a memory overclock so I just use MSI Afterburner (Since there is also no switches for FAN control in lol so you kinda have to use afterburner anyways)

**Comment by eaamgh** [comment]
Yes, --dualdevices doesn't work with DUALTON mode, if  I remember it correctly. Did the same thing mohsenk94 suggested: two miners. lolminer for GPU's running on DUALTON mode and another miner for the other GPU's, both with a "device list"  switch to select which ones would run dual mining and the ones that would not.

**Reply by cypher619** [reply]
--maxdualimpact does this not --dualdevices see Guide Tuning Dual Mining in the Wiki section
